connection.go

```diff
@@ -69,11 +69,14 @@ type ConnectionBuilder struct {
 	password     string
 	tokens       []string
 	scopes       []string
+	transport    WrappedTransport
```

Comment:
As this is not really a transport I think it would be better to name it `transportWrapper`. And the type should also be `TransportWrapper`.

Reply:
Fixed